velox/functions/prestosql/aggregates/SumDataSizeForStatsAggregate.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/Aggregate.h"
+#include "velox/expression/FunctionSignature.h"
+#include "velox/functions/lib/aggregates/SimpleNumericAggregate.h"
+#include "velox/functions/prestosql/aggregates/AggregateNames.h"
+#include "velox/serializers/PrestoSerializer.h"
+
+using namespace facebook::velox::functions::aggregate;
+
+namespace facebook::velox::aggregate::prestosql {
+
+namespace {
+std::unique_ptr<VectorSerde>& getVectorSerde() {
+  static std::unique_ptr<VectorSerde> serde =
+      std::make_unique<serializer::presto::PrestoVectorSerde>();
+  return serde;
+}
+
+class SumDataSizeForStatsAggregate
+    : public SimpleNumericAggregate<int64_t, int64_t, int64_t> {
+  using BaseAggregate = SimpleNumericAggregate<int64_t, int64_t, int64_t>;
```

Comment:
Move using BaseAggregate = under public section or private section if only used internally. Note by default, it is private